Mathlib/LinearAlgebra/CliffordAlgebra/Contraction.lean

```diff
@@ -241,22 +242,21 @@ local infixl:70 "⌊" => contractRight
 
 /-- Auxiliary construction for `CliffordAlgebra.changeForm` -/
 @[simps!]
-def changeFormAux (B : BilinForm R M) : M →ₗ[R] CliffordAlgebra Q →ₗ[R] CliffordAlgebra Q :=
+def changeFormAux (B : M →ₗ[R] M →ₗ[R] R) : M →ₗ[R] CliffordAlgebra Q →ₗ[R] CliffordAlgebra Q :=
   haveI v_mul := (Algebra.lmul R (CliffordAlgebra Q)).toLinearMap ∘ₗ ι Q
-  v_mul - contractLeft ∘ₗ (BilinForm.toLin B)
+  v_mul - contractLeft ∘ₗ B
 #align clifford_algebra.change_form_aux CliffordAlgebra.changeFormAux
 
-theorem changeFormAux_changeFormAux (B : BilinForm R M) (v : M) (x : CliffordAlgebra Q) :
+theorem changeFormAux_changeFormAux (B : M →ₗ[R] M →ₗ[R] R) (v : M) (x : CliffordAlgebra Q) :
```

Comment:
@eric-wieser Should I put this back to `(LinearMap.)BilinForm R M`?

Reply:
Yes please

Reply:
@eric-wieser okay, I think I've done that everywhere now. Thanks.